### DISCUSSION TITLE
Preloading Images & Font in NextJS -SSG

### DISCUSSION THREAD

**Discussion by Vatsal2k** [vercel/next.js#15098] [Help]
We are using NextJS 9.3.2 Framework with Static Site Generator i.e SSG with the introduction of Google Lighthouse 6 Largest Contentful paint (LCP)is affecting my Site performance score . Most of sites have a Hero Image in above the fold content.

SO i am looking to Preload the image to cut down the time of LCP. Can you guys guide how can i preload big Hero image in NextJS with SSG

**Comment by mulholo** [comment]
https://github.com/cyrilwanner/next-optimized-images is a plugin that will get you a lot of image performance benefits and help improve your lighthouse scores.
